### ISSUE TITLE
Webapp:staticwebapp - create defaults are invalid

### ISSUE THREAD

**Issue by anthonychu** [Azure/azure-cli#20830]
**Describe the bug**

`az staticwebapp create` currently defaults to the following locations:
https://github.com/Azure/azure-cli/blob/86ffba42f91a9082743237286d63d89ef6e76ce4/src/azure-cli/azure/cli/command_modules/appservice/static_sites.py#L343

`output_location` is incorrect. It should default to empty string.

I think the app and api locations should probably default to empty strings as well. @miwebst @simonaco What do you think?

**To Reproduce**

Run `az staticwebapp create` without supplying `--output-location`.

**Expected behavior**

The workflow file created should have an empty `output_location`.

/cc @StrawnSC

**Comment by yonzhan** [comment]
route to service team

**Comment by StrawnSC** [comment]
@anthonychu I've got a draft [PR](https://github.com/Azure/azure-cli/pull/20955) in the works for this. The PR also includes an unrelated feature request from @simonaco to reduce the number of properties shown on SWA objects returned from the CLI.

I went ahead and changed the default `output_location` value to be `None`, but have not yet changed the defaults for `app_location` and `api_location`. Based on the portal defaults, it seems like `app_location` should default to `/` and `api_location` should default to `None` (see below). 

@anthonychu should I just use the same defaults as the portal for `app_location` and `api_location`? 

<img width="864" alt="Screen Shot 2022-01-12 at 11 34 40 AM" src="https://user-images.githubusercontent.com/10121214/149209323-8886ec2f-9480-4769-9036-3a687b951de3.png">
